tests/server.py

```diff
@@ -523,6 +524,27 @@ def add_tcp_client_callback(
         """
         self._tcp_callbacks[(host, port)] = callback
 
+    def connectUNIX(
+        self,
+        address: str,
+        factory: ClientFactory,
+        timeout: float = 30,
+        checkPID: int = 0,
+    ) -> IConnector:
+        """Unix sockets aren't supported for unit tests yet. Make it obvious to any developer trying it out that they will need to do some work before being able to use it in tests."""
```

Comment:
(wrap line)

Reply:
Fixed in [2f20dda](https://github.com/matrix-org/synapse/pull/15708/commits/2f20ddae4991ce5521822baefa5bb8630c9d220f)

(Duplicated in function just below, since its related)